io/post_gfs.F90

```diff
@@ -1161,6 +1161,7 @@ subroutine set_postvars_gfs(wrt_int_state,mpicomp,setvar_atmfile,   &
                   endif
                 enddo
               enddo
+!              print *,'in gfs_post, get tisfc=',maxval(ti), minval(ti)
```

Comment:
is this line needed?

Reply:
Will be removed  later.

Reply:
Removed this line in NCAR's dtc/develop. Will come to EMC develop with our next commit.